networkit/cpp/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+networkit_add_module(globals
+    GlobalState.cpp
+    )
```

Comment:
There is already a `global` module. Let's name this `globalstate`, not `globals`.